git_build_branch/branch_builder.py

```diff
@@ -324,24 +327,25 @@ def main():
     parser.add_argument("--push", action="store_true", help="Push the changes to remote git repository.")
     args = parser.parse_args()
 
+    with open(args.config_path) as config_yaml:
+        config_raw = yaml.safe_load(config_yaml)
+    repo_config = BranchConfig.wrap(config_raw)
+    trunk = repo_config.trunk or "master"
```

Comment:
Nit: could you do this in `BranchConfig` itself?  Like 
```python
    trunk = jsonobject.StringProperty(default="master")
```

Reply:
Thanks, updated.